boot/freeldr/freeldr/ntldr/winldr.c

```diff
@@ -1278,6 +1278,18 @@ LoadAndBootWindowsCommon(
     WinLdrpDumpArcDisks(LoaderBlockVA);
 #endif
 
+    /* Show a notice to user about debugger connection */
+    if (!strstr(LoaderBlockVA->LoadOptions, "CRASHDEBUG") &&
+        !strstr(LoaderBlockVA->LoadOptions, "NODEBUG") &&
+        strstr(LoaderBlockVA->LoadOptions, "DEBUG"))
+    {
+        MachVideoPuts("Debug mode is enabled.\n"
+                      "If you're stuck here it means your debugger is not attached.\n"
```

Comment:
Could you give detailed steps to reproduce the issue? (VM, Build, Options, ...)

Reply:
According to Simone, this should happen when you boot "ReactOS (Debug)" on a VM with a hypervisor like VirtualBox. On QEMU there's a wait for like 5 seconds but it will boot automatically even if you don't connect a debugger, and the reason seems to be broken serial implementation in QEMU.

Reply:
Thanks. Could you give the full details (livecd or bootcd, gcc or msvc, x86 or amd64, debug or release, which com ports are present, which HAL is used, ...)?
If possible, how does it behave on RealHW?
https://reactos.org/wiki/File_Bugs